server/src/resolvers/user.resolver.ts

```diff
@@ -83,24 +98,25 @@ export class UserResolver {
 	 */
 	@Mutation(() => UserResponse)
 	async login(
-		@Arg('options') options: UsernamePasswordInput,
+		@Arg('options') options: LoginInput,
 		@Ctx() { prisma, req }: Context,
 	): Promise<UserResponse> {
 		// Pulls the username and password off of the options
-		const { username, password } = options;
+		const { credential, password } = options;
 
 		// Attempts to find a user with the passed in username
-		const user = await prisma.user.findUnique({
-			where: { username },
+		const user = await prisma.user.findFirst({
+			where: { OR: [{ username: credential }, { email: credential }] },
 		});
 
 		// Sends an error if the username wasn't found
 		if (!user) {
 			return {
 				errors: [
 					{
-						field: 'username',
-						message: `Could not find a user with the username ${username}`,
+						field: 'credential',
```

Comment:
This is kinda awkward because it forces string matching on the front end for the credential input field in order to get the error